core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -2294,6 +2325,13 @@ private long loadMetadataAndJournal(LockedInodePath inodePath, LoadMetadataOptio
   }
 
   /**
+   * Status of UFS path for which loading metadata.
+   */
+  public enum LoadMetadataStatus {
```

Comment:
Would it be cleaner to directly pass the `UnderFileStatus` or introduce different methods?